WidgetExtension/Variants/Hashtag/HashtagWidgetView.swift

```diff
@@ -0,0 +1,134 @@
+                }
+                HStack(spacing: 0) {
+                    Image(systemName: "star")
+                        .foregroundColor(.secondary)
+                    Text("\(entry.hashtag.favoriteCount)")
+                        .font(.caption)
+                        .foregroundColor(.secondary)
+                }
+                Text(entry.hashtag.hashtag)
+                    .font(.caption)
+                    .fontWeight(.semibold)
+                    .foregroundColor(.secondary)
+            }
+        }
+        .padding(EdgeInsets(top: 12, leading: 29, bottom: 12, trailing: 29))
+    }
+
+    private func viewForRectangularAccessory() -> some View {
+        VStack(alignment: .leading, spacing: 1) {
+            HStack(alignment: .center, spacing: 3) {
+                Image("BrandIcon")
+                    .renderingMode(.template)
+                    .resizable()
+                    .frame(width: 10, height: 10)
+                    .foregroundColor(.secondary)
+                Text("|")
+                    .font(.system(size: UIFontMetrics.default.scaledValue(for: 12)))
+                    .foregroundColor(.secondary)
+                Text(entry.hashtag.hashtag)
+                    .font(.system(size: UIFontMetrics.default.scaledValue(for: 13)))
+                    .fontWeight(.heavy)
+                    .foregroundColor(.secondary)
+            }
+            Text(statusHTML: entry.hashtag.content, fontSize: 11, fontWeight: 510)
+                .lineLimit(3)
+        }
+    }
+}
+
+/// Inspired by: https://swiftuirecipes.com/blog/swiftui-text-with-html-via-nsattributedstring
+extension Text {
+    init(statusHTML htmlString: String, fontSize: Int = 16, fontWeight: Int = 400, colorScheme: ColorScheme = .light) {
+
+        let textColor = (colorScheme == .light) ? "#000000" : "#FFFFFF"
+        let accentColor = (colorScheme == .light) ? "#563ACC" : "#858AFA"
+
+        let fullHTML = """
+<!doctype html>
+<html>
+    <head>
+        <style>
+                body {
+                    font-family: -apple-system;
+                    font-size: \(fontSize)px;
+                    font-weight: \(fontWeight);
+                    line-height: 133%;
+                    color: \(textColor);
+                }
+
+                a {
+                    color: \(accentColor);
+                }
+            }
+        </style>
+    </head>
+    <body>
+        \(htmlString)
+    </body>
+  </html>
+"""
+
+        let attributedString: NSAttributedString
+        if let data = fullHTML.data(using: .unicode),
+           let attrString = try? NSAttributedString(data: data,
+                                                    options: [.documentType: NSAttributedString.DocumentType.html],
+                                                    documentAttributes: nil) {
+            attributedString = attrString
+        } else {
+            attributedString = NSAttributedString()
```

Comment:
Not great UX but better than blank IMO

```suggestion
            attributedString = NSAttributedString(string: htmlString)
```